### PR TITLE
Chloe feature profile

### PR DESCRIPTION
**Pull request by Chloe-Meinshausen**
Hi, In this PR 

- I do a bunch of error handling, 

-   fix the routes so live data works
- update some styling
- update routes so they all work
- update profile page

### CODE REVIEW

File: server/controllers/account.controllers.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ module.exports = {
       // console.log('LISTINGS');
       models.listings.get(owner_id, (err, data) => {
         if (err) {
-          res.status(500).send(err);
+          console.log('LISTINGS ERROR', err);
+          res.status(500).send(err, err);
```
Comment (nickgerrard) [inline server/controllers/account.controllers.js]: super minor, but you are sending err twice!

File: client/src/account/StackVersionListings.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import ListGroup from 'react-bootstrap/ListGroup';
+import React, {useState, useEffect} from 'react';
+import Table from 'react-bootstrap/Table';
+import Container from 'react-bootstrap/Container';
+import { BsSortDownAlt, BsSortUpAlt } from 'react-icons/bs';
+import {Link } from 'react-router-dom';
```
Comment (nickgerrard) [inline client/src/account/StackVersionListings.jsx]: spacing missing

File: client/src/account/StackVersionListings.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+import moment from 'moment';
+import RefundButton from '../checkout/RefundButton.jsx';
+import Image from 'react-bootstrap/Image';
+import Stack from 'react-bootstrap/Stack';
+import '../App.css';
+import './accountStyles.css';
+import noImagePhoto from '../itemView/samplePhotos/unavailable-image.jpeg';
+
+export default function StackVersionListings (props) {
+  props.column;
```
Comment (nickgerrard) [inline client/src/account/StackVersionListings.jsx]: not quite sure what this line does?

File: server/controllers/account.controllers.js (hunk 1 above)
Comment (Chloe-Meinshausen) [reply server/controllers/account.controllers.js]: fixed!

File: client/src/account/StackVersionListings.jsx (hunk 2 above)
Comment (Chloe-Meinshausen) [reply client/src/account/StackVersionListings.jsx]: How do you spot these things!  you are a amazing

File: client/src/account/StackVersionListings.jsx (hunk 3 above)
Comment (Chloe-Meinshausen) [reply client/src/account/StackVersionListings.jsx]: it doesn't!

**Pull request merged**